scripts/leapp_script.py

```diff
@@ -355,17 +365,15 @@ def parse_results(output, reboot_required=False):
     output.report = report_txt
 
 
-def update_insights_inventory():
+def update_insights_inventory(output):
     """Call insights-client to update insights inventory."""
     print("Updating system status in Red Hat Insights.")
-    output, returncode = run_subprocess(["/usr/bin/insights-client"])
+    _, returncode = run_subprocess(cmd=["/usr/bin/insights-client"])
 
     if returncode:
-        raise ProcessError(
-            message="Failed to update Insights Inventory by registering the system again. See output the following output: %s"
-            % output,
-            report="insights-client execution exited with code '%s'." % returncode,
-        )
+        print("System registration failed with exit code %s." % returncode)
+        output.message += " Failed to update Insights Inventory."
```

Comment:
should we also add the `alert: true` ? I don't know if it's that important, but UI would show just Success and plain text and it would look like complete sucess :thinking: